### PR TITLE
Fix issue #735 & #736 for visual updates

### PR DESCRIPTION
**Pull request by youwenliang**
Fix issue #735 & #736 for visual updates

### CODE REVIEW

File: data/panel.css [hunk 1]
```diff
@@ -795,12 +801,12 @@ canvas {
     border-radius: 4px;
     padding: 5px;
 
-    font-size: 2vw;
+    font-size: 12px;
```
Comment (meandavejustice) [inline data/panel.css]: I'd rather have a solution that scales to different sizes

Comment (meandavejustice) [inline data/panel.css]: May have to use media queries with a minimum size

File: data/icon-overlay.js [hunk 2]
```diff
@@ -240,12 +240,15 @@ function getTemplate() {
 
   containerEl.className = 'minvid__overlay__container';
   playIconEl.className = 'minvid__overlay__icon';
+  playIconEl.id = 'minvid__overlay__icon__play';
   playIconEl.title = 'Play Now';
-  addIconEl.className = 'minvid__overlay__icon__add';
+  addIconEl.className = 'minvid__overlay__icon';
+  addIconEl.id = 'minvid__overlay__icon__add';
```
Comment (meandavejustice) [inline data/icon-overlay.js]: Neither of the issues linked reference overlay icon, so I'm not sure what these changes are addressing.

File: data/panel.css (hunk 1 above)
Comment (youwenliang) [inline data/panel.css]: Agree, that was a temporary test to see the font-size differences.

File: data/icon-overlay.js (hunk 2 above)
Comment (youwenliang) [inline data/icon-overlay.js]: This was for the individual icon to have hover state, check the Gif below:
apr-15-2017 00-30-12

Comment (meandavejustice) [inline data/icon-overlay.js]: Nice! Is there an issue for this one?

Comment (youwenliang) [inline data/icon-overlay.js]: I think it was part of #737 but was considered as a duplicate.

Comment (meandavejustice) [inline data/icon-overlay.js]: So does this take care of #439?

Comment (youwenliang) [inline data/icon-overlay.js]: Not quite, I removed the fade effect so as long as the mouse is hovered on a video to overlay icons won't fade away. 

However, the description in #439 suggests handling the idle situation where the overlay icons still fade away but will reappear when the mouse is moving (without re-hovering).

**Comment by meandavejustice** [comment]
@youwenliang happy to merge once the conflict is resolved.

**Pull request merged**